machine/accton/accton_as4510_52t/kernel/platform-as4510_52t.patch

```diff
@@ -0,0 +1,481 @@
+Support for Accton AS4510_52T platform (ARM Cortex-A9 / Hurricane2) networking platform
+Baseline files sourced from Broadcom LDK 3.4.10
+
+diff --git a/arch/arm/boot/dts/accton_as4510_52t.dts b/arch/arm/boot/dts/accton_as4510_52t.dts
+new file mode 100644
+index 0000000..5ef7c60
+--- /dev/null
++++ b/arch/arm/boot/dts/accton_as4510_52t.dts
+@@ -0,0 +1,92 @@
++/*
++ * Accton AS4510_52T Device Tree Source
++ *
++ * Copyright 2015, Accton Technology Corporation.
++ *
++ * This program is free software; you can redistribute  it and/or modify it
++ * under  the terms of  the GNU General  Public License as published by the
++ * Free Software Foundation;  either version 2 of the  License, or (at your
++ * option) any later version.
++ *
++ */
++/dts-v1/;
++/include/ "helix4.dtsi"
++
++/ {
++    model = "accton,as4510_52t";
++    compatible = "accton,as4510_52t";
++
++    aliases {
++        serial0 = &uart0;
++    };
++
++    memory {
++        reg = <0x61000000 0x7f000000>;
```

Comment:
I think you fixed this the wrong way round -- I would have corrected the .dts file.  I would not artificially reserve 0x0100_0000 in the u-boot configuration.  That is just wasting 16MB of SDRAM.